### ISSUE TITLE
Remove CenterCrop during testing

### ISSUE THREAD

**Issue by naga-karthik** [ivadomed/model_sc_centerline#1]
As suggested in this [comment](https://github.com/neuropoly/idea-projects/issues/15#issuecomment-1402067311), the CenterCrop parameters used during training might not work for test images as some might be larger than that. To avoid this, it is best to remove center cropping for testing and check if we still get good predictions. 

TODO: 
- [x] Disable CenterCrop during testing and report findings here

**Comment by naga-karthik** [comment]
It appears that CenterCrop _must_ be used during inference, because without it, we run into [this](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/loader/mri3d_subvolume_segmentation_dataset.py#L148) RuntimeError. This is because for inputs with variable sizes, we cannot evenly divide the input volume into multiple sub-volumes hence raising this error. 

I expected this error because dealing with variable input sizes during testing is a common problem and CenterCrop is essentially used to make sure that all inputs are uniformly-sized before feeding it to the model for inference.

**Comment by jcohenadad** [comment]
> It appears that CenterCrop _must_ be used during inference

Not necessarily. We've successfully explored inputting larger images at test time, see for example:
- https://github.com/ivadomed/ivadomed/issues/1049
- https://github.com/ivadomed/ivadomed/issues/1255
- https://github.com/ivadomed/ivadomed/pull/1101

Additional digging would be welcome.
